### PR TITLE
chore(CI): rename arch package name

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

File: build [hunk 1]
```diff
@@ -28,6 +28,27 @@ elif [ "$(uname -s)" = 'Linux' ]; then
...
+    aarch64)
+        ARCH=arm64
+        ;;
+    arm*)
+        ARCH=arm
+        ;;
+    *)
+        ARCH=$(uname -m)
+esac
+export ARCH
```
Comment (zmstone) [inline build:50]: I think it's more clear if we use this 'rewrite' logic:
```suggestion
ARCH="$(uname -m)"
case "$ARCH" in
    x86_64)
        ARCH='amd64'
        ;;
    aarch64)
        ARCH='arm64'
        ;;
esac
export ARCH
```

File: deploy/packages/rpm/Makefile [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Keep this short to avoid bloating beam files with long file path info
```
Comment (zmstone) [inline deploy/packages/rpm/Makefile:1]: this comment is for line 3.
add a new line in between is confusing.

File: build [hunk 3]
```diff
@@ -28,6 +28,20 @@ elif [ "$(uname -s)" = 'Linux' ]; then
...
+ARCH="$(uname -m)"
+case "$ARCH" in
+    x86_64)
+        ARCH='amd64'
+        ;;
+    aarch64)
+        ARCH='arm64'
+        ;;
+    arm*)
+        ARCH=arm
```
Comment (Rory-Z) [inline build:40]: @zmstone  I want to rename `armv6`, `armv7`, `armhf` to `arm`, what do you think?

**Review by tigercl — APPROVED** [review]

File: build (hunk 3 above)
Comment (zmstone) [reply build:40]: if `SYSTEM` can tell them apart, it's fine.

**Pull request merged**